### PR TITLE
Add myself and dims as reviewers & sig-testing leadership as approvers

### PR DESCRIPTION
**Pull request by upodroid**

### CODE REVIEW

**Comment by dims** [comment]
yes please!

**Comment by michelle192837** [comment]
Ah sorry, OWNERS in test-infra aren't quite up to date. I'd use https://github.com/kubernetes/community/blob/master/sig-testing/README.md#leadership as a reference instead.

**Comment by upodroid** [comment]
Fixed

**Comment by michelle192837** [comment]
Thanks! (Sorry to nitpick again but I think just having the chairs atm instead of all the leads might be better; we can stay on temporarily while we help find other owners, but unless any particular leads want to add themselves I don't think they need to also find owners).

Otherwise looks good to me, thanks!

**Review by michelle192837 — APPROVED** [review]
/hold

For others to take a look at.

**Comment by upodroid** [comment]
/hold cancel

I limited approvers to chairs only for now.

**Review by michelle192837 — APPROVED** [review]
Looks good, thanks again! ^^

File: OWNERS
Comment (michelle192837) [reply OWNERS:11]: Ah, sorry about that, noted for future. I was under the impression chair should help facilitate subproject management but on another look it does look more in the TL area. (I'd like to say we wouldn't repeat this but Boskos is probably in a similar boat for ownership 😅 )

I'll stick on "we should help find more active owners here" for this case in particular and shift to more TL involvement in future.

Comment (BenTheElder) [reply OWNERS:11]: I guess my point is we shouldn't add them "because they're chairs", if we're going to do that it should be TLs, but I'd prefer we do neither. TLs should be facilitating things like creating/removing projects but deferring to subproject owners for day to day PR approval except possibly to help resolve a disagreement amongst subproject technical owners.

If chairs want to facilitate by helping revive dead projects that's fine, but we shouldn't put `# chair` and give the impression that chairs are the technical owners of the SIG, we want to encourage folks to help with the chair role even if they aren't deep on the technicals of a SIG yet.

And agree, we really should try to find some owners for the specific projects as opposed to having to add either of the SIG-level lead groups.

Comment (michelle192837) [reply OWNERS:11]: That's fair, yeah. I do want to help temporarily and stay accountable for finding/encouraging more owners if that makes sense for kubetest2. I can update this with a TODO to clarify that instead of the `# chair` comment (and add an end date on it?)

Comment (BenTheElder) [reply OWNERS:11]: That's fine, and I'm relatively unconcerned with this instance, I just don't want to start a pattern confusing the chair role, it's been tough to get SIGs to do chair/TL split 😅

**Pull request merged**
